web/styles/app_variables.css

```diff
@@ -175,12 +175,14 @@
     --color-search-dropdown-top-border: hsla(0deg 0% 0% / 10%);
     --color-background-image-loader: hsl(0deg 0% 0% / 10%);
     --color-icon-purple: hsl(240deg 35% 60%);
-    --color-background-dropdown-menu: hsl(0deg 0% 100%);
-    --color-border-popover-menu: hsl(0deg 0% 0% / 10%);
+    --color-background-popover-menu: hsl(0deg 0% 100%);
+    --color-border-popover-menu-seperator: hsl(0deg 0% 0% / 10%);
```

Comment:
_separator_ is so spelled, with an _a_

Reply:
Sorry! Fixed the typo and added spell check to my workflow to prevent such inconveniences in the future.